internal/slicer/report_test.go

```diff
@@ -200,22 +207,63 @@ var reportTests = []struct {
 	add: []sliceAndEntry{
 		{entry: fsutil.Entry{Path: "/file"}, slice: oneSlice},
 	},
-	err: `cannot add path "/file" outside of root "/base/"`,
+	err: `cannot add path: "/file" outside of root "/base/"`,
+}, {
+	summary: "Error for mutated path outside root",
+	mutate:  []*fsutil.Entry{{Path: "/file"}},
+	err:     `cannot mutate path: "/file" outside of root "/base/"`,
 }, {
 	summary: "File name has root prefix but without the directory slash",
 	add: []sliceAndEntry{
 		{entry: fsutil.Entry{Path: "/basefile"}, slice: oneSlice},
 	},
-	err: `cannot add path "/basefile" outside of root "/base/"`,
+	err: `cannot add path: "/basefile" outside of root "/base/"`,
+}, {
+	summary: "Add mutated regular file",
+	add: []sliceAndEntry{
+		{entry: sampleFile, slice: oneSlice},
+		{entry: sampleDir, slice: oneSlice},
+	},
+	mutate: []*fsutil.Entry{&sampleFileMutated},
+	expected: map[string]slicer.ReportEntry{
+		"/exampleDir/": {
+			Path:   "/exampleDir/",
```

Comment:
Aren't our sample files all lowercased and dashed in other packages?  Not important, but would be nice to have a single convention.

Reply:
Changed the ones in this package to follow the convention in https://github.com/canonical/chisel/pull/131/commits/30a19cfcb811c1d7075309f2975766d8df3f1668.